automated install/basic-install.sh

```diff
@@ -702,6 +716,30 @@ find_IPv4_information() {
     IPV4_ADDRESS=$(ip -oneline -family inet address show | grep "${IPv4bare}/" | awk '{print $4}' | awk 'END {print}')
 }
 
+confirm_ipv6_only() {
+    # Confirm from user before IPv6 only install,
+    # unless PI_HOLE_IPV6_ONLY environment variable is true,
```

Comment:
```suggestion
    # unless PI_HOLE_IPV6_ONLY environment variable is true,
```

Reply:
reverence to removed variable removed in new commit